TwipsterApp/Models/User.cs

```diff
@@ -22,5 +23,7 @@ public class User
         [Required]
         [MinLength(3), MaxLength(15)]
         public string Password { get; set; }
+
+        public ICollection<Post> Posts { get; set; } = new List<Post>();
```

Comment:
Mark this field as virtual. Remove initialisation (This is initialisation is redundant in ef using context).